content/technotes/2018-09-09-github-badges.md

```diff
@@ -0,0 +1,395 @@
+---
+slug: "github-badges"
+title: "All the Badges One Can Earn: Parse Badges of CRAN Packages READMEs"
+date: 2018-09-09
+authors:
+  - name: Maëlle Salmon
+    url: https://masalmon.eu/
+categories:
+  - technotes
+topicid: 
+tags:
+- Codemeta
+- codemetar
+- Markdown
+- xml2
+- commonmark
+output: 
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+A while ago we
+[onboarded](https://github.com/ropensci/onboarding/issues/130) an
+exciting package, [`codemetar`](https://github.com/ropensci/codemetar/)
+by [Carl Boettiger](https://ropensci.org/about/#team). `codemetar` is an
+R specific information collector and parser for the [CodeMeta
+project](https://github.com/codemeta). In particular, `codemetar` can
+digest metadata about an R package in order to fill [the terms
+recognized by CodeMeta](https://codemeta.github.io/terms/). This means
+extracting information from DESCRIPTION but also from e.g. continuous
+integration badges in the README! In this note, we’ll take advantage of
+`codemetar::extract_badges` function to explore the diversity of badges
+worn by the READMEs of CRAN packages.
+
+Why `codemetar::extract_badges`, and how
+----------------------------------------
+
+CodeMeta recognized terms include information we’ve been getting from
+badges:
+
+-   “contIntegration”, URLs to continuous integration services such as
+    Travis, Appveyor, Codecov;
+
+-   “review”, information about review of the software if there was one.
+    `codemetar` recognizes information from the peer-review badge we add
+    to the README of onboarded packages thanks to work by [Karthik
+    Ram](https://ropensci.org/about/#team).
+
+The list might get longer, so instead of using regular expressions on
+the README text, we extract and memoize all badges at once to a
```

Comment:
some readers may not know what you mean by `memoize`, I think?

Reply:
Yes, and I'll also add a link to our dev guide CI section since even CI could be new to some readers